shared/constants/engine.js

```diff
@@ -32,13 +34,11 @@ function _sagaWaitingDecorator(rpcNameKey, saga, waitingAction) {
     if (waitingAction) {
       yield Saga.put(waitingAction(false))
     }
-    yield Saga.put(EngineGen.createWaitingForRpc({name: rpcNameKey, waiting: false}))
```

Comment:
this was just used by login and its not anymore so we can entirely remove the engine store